src/dataStructures.hpp

```diff
@@ -371,8 +393,9 @@ struct ChemistryInput {
   double reactantStoich[gpudata::MAXSPECIES * gpudata::MAXREACTIONS];
   double productStoich[gpudata::MAXSPECIES * gpudata::MAXREACTIONS];
   ReactionModel reactionModels[gpudata::MAXREACTIONS];
-  double reactionModelParams[gpudata::MAXCHEMPARAMS * gpudata::MAXREACTIONS];
+  // double reactionModelParams[gpudata::MAXCHEMPARAMS * gpudata::MAXREACTIONS];
```

Comment:
Delete? (b/c this functionality is replaced by `ReactionInput` right?)

Reply:
Deleted.